index.html

```diff
@@ -14,183 +14,7 @@
-        <div class="hamburger-icon">
-          <img src="./images/menu-hamburger.svg" alt="" onclick="openMenu()" />
-        </div>
-        <div class="menu" id="menu">
-          <ul>
-            <li><a href="#">Features</a></li>
-            <li><a href="#">Team</a></li>
-            <li><a href="#">Sign In</a></li>
-          </ul>
-        </div>
-      </nav>
-    </header>
-
-    <main>
-      <section class="hero">
-        <div class="hero-img">
-          <img src="/images/illustration-1.svg" alt="illustration-1" />
-        </div>
-        <div class="hero-content">
-          <h1>All your files in one secure location, accessible anywhere.</h1>
-          <p>
-            Fylo stores your most important files in one secure location. Access
-            them wherever you need, share and collaborate with friends, family,
-            and co-workers.
-          </p>
-          <div class="hero-content-input-btn">
-            <input
-              type="email"
-              name="email"
-              id="email"
-              placeholder="Enter your email"
-            />
-            <button class="hero-content-btn">Get Started</button>
-          </div>
-        </div>
-      </section>
-
-      <section class="section-two">
-        <div class="section-two-img">
-          <img src="./images/illustration-2.svg" alt="illustration-2" />
-        </div>
-        <div class="section-two-container">
-          <div class="section-two-content">
-            <h3>Stay productive, wherever you are.</h3>
-            <p>
-              Lorem ipsum dolor sit amet consectetur adipisicing elit. Obcaecati
-              voluptates hic facilis iste deserunt ex velit, cumque optio enim,
-              sint quaerat tempora magni? Minima illum, itaque optio distinctio
-              nostrum aut.
-            </p>
-            <p>
-              Lorem ipsum dolor sit amet consectetur adipisicing elit. Obcaecati
-              voluptates hic facilis iste deserunt ex velit, cumque optio enim,
-              sint quaerat tempora magni? Minima illum, itaque optio distinctio
-              nostrum aut.
-            </p>
-          </div>
-          <div class="section-two-link">
-            <p>See how Fylo works</p>
-            <img class="arrow" src="./images/icon-arrow.svg" alt="arrow" />
-          </div>
-          <div class="blackquoto">
-            <blockquote class="blockquote">
-              Lorem ipsum dolor sit amet consectetur adipisicing elit.
-              Exercitationem, nobis consectetur. Laborum deserunt excepturi
-              natus, reiciendis repellendus, veritatis voluptas animi sequi
-              quasi quae fugiat soluta similique tenetur? Reiciendis,
-              voluptatum? Tempora.
-            </blockquote>
-          </div>
-        </div>
-      </section>
-
-      <section>
-        <div class="section-three">
-          <div class="section-three-content">
-            <h1>Get early access today</h1>
```

Comment:
I think this would be best as an h2 or h3. You already have an h1 in the hero which is larger than this in the design